modules/programs/nushell.nix

```diff
@@ -121,6 +121,16 @@ in {
         this option) to command strings or directly to build outputs.
       '';
     };
+
+    envVars = mkOption {
```

Comment:
Personally I'd write this out as environmentVariables. Just to be in line with options like sessionVariables.

Reply:
you suggestion is much better indeed.